browsers/opera_android.json

```
@@ -18,13 +18,14 @@
           "engine_version": "2.7"
         },
         "11.1": {
-          "release_date": "2011-06-30",
+          "release_date": "2011-04-11",
           "release_notes": "https://dev.opera.com/blog/opera-mobile-11-1-new-features-and-additions/",
           "status": "retired",
           "engine": "Presto",
           "engine_version": "2.8"
         },
         "11.5": {
+          "release_date": "2011-06-28",
```

I couldn't find any citation for this date. Between this press release and some searches, I propose changing this to October 12 2011

- https://web.archive.org/web/20111110083054/http://www.opera.com/press/releases/2011/11/02/
- https://www.google.com/search?q=%22opera%22+%22android%22+%2211.5%22

```suggestion
          "release_date": "2011-10-12",
```